### DISCUSSION TITLE
[FEATURE REQUEST]:  Support for regression models

### DISCUSSION THREAD

**Discussion by bolcman** [dotnet/spark#778] [Q&A] [answered]
Hi, 

I see that some of the MLlib functionalities are already in place, is there any plan to support other?
Specifically, I'm interested in regression models.

Thanks, 
Aleks

**Comment by GoEddie** [comment, marked as answer]
Hi,

We have started implementing the ml features:

https://github.com/dotnet/spark/issues/381

There isn’t a formal plan to implement ml other than I started with features and was then going to move onto the rest of the ml objects.

If you would like to contribute and would like some pointers I would be happy to help. If you aren’t able to commit but are blocked by an object not being available let me know and I can help implement it for you.
